src/VisualStudio/Core/Def/Implementation/ColorSchemes/ColorSchemeApplier.ForegroundColorDefaulter.cs

```diff
@@ -259,6 +259,13 @@ public void DefaultClassifications()
                     return;
                 }
 
+                // If the theme is customized, then do not change the classification colors.
+                if (isThemeCustomized)
+                {
+                    _settings.HasThemeBeenDefaulted[themeId] = true;
```

Comment:
it seems a little odd that `isThemeCustomized` makes `HasThemeBeenDefaulted` as true. Should we remove this if and just add a check in the if above? Does this setting need to be set?

Reply:
There is no reason to try and default colors in the future either.

Reply:
Yea, but `HasThemeBeenDefaulted` seems misleading in this case. Maybe we should rename that? Idk if it's a breaking change, but it just seems weird when looking and could lead to future bugs if the intent of the setting is misunderstood.

Reply:
What if I rename it all to "SkipThemeDefaulting"